guardian/support-frontend
# Create Newspaper archive benefit Thank you module structure

## What are you doing in this PR?

Implementing the structure and logic for the new Thank You module: Newspaper archive benefit. This re-uses the existing module styling but updating the placement based on new designs.



[**Trello Card**](https://trello.com/c/domGTnMJ/1018-ux-design-of-newspaper-archive-component-on-thank-you-page)

## Why are you doing this?

Creating the initial design structure for the new benefit.



## How to test

Activate AB test `newspaperArchiveBenefit` - this module is shown regardless of which variant is in play. Go through to Tier 3 thank you page, see new module with temp imagery, new images not yet provided. 

Storybook should catch that there are no regressions to existing thank you modules. (Although we aren't snapshotting the mobile versions so I've tested manually)


## Screenshots
Desktop
![image](https://github.com/user-attachments/assets/e68f46d2-d470-4a8a-8a37-bfca48d5a1d0)

Mobile
![image](https://github.com/user-attachments/assets/1ca13ae6-957d-409d-938e-5531ce917277)
